### DISCUSSION TITLE
DRAMSim2 vs. DRAMSim3

### DISCUSSION THREAD

**Discussion by colinrioux** [gem5/gem5#746] [gem5-dev]
Short question:

I am not too familiar with gem5's history, but why is DRAMsim2 still supported? Is it because of backwards compatability? I noticed in the ext/DRAMsim3 README DRAMsim3 seems required to build gem5, whereas DRAMsim2 is not required.

**Comment by powerjg** [comment]
1. The DRAMSim2 integration's code is still around, but it's untested.
2. DRAMSim3 is not *required* for gem5. We support DRAMSim3 as an optional DRAM model, but it's not the main or suggested DRAM model. We encourage you to use gem5's built-in DRAM model, which is nearly as accurate and DRAMSim, but much faster. See https://web.eecs.umich.edu/~twenisch/papers/ispass14.pdf for details.
